application/src/main/java/org/togetherjava/tjbot/features/moderation/history/Duration.java

```diff
@@ -0,0 +1,25 @@
+package org.togetherjava.tjbot.features.moderation.history;
+
+/**
+ * Holds duration constants used for both purge history command and message history routine.
+ */
+public enum Duration {
+    // All new/existing options for duration in PurgeHistoryCommand should be within max and min
+    // duration
+    PURGE_HISTORY_MAX_DURATION(24),
+    PURGE_HISTORY_MIN_DURATION(1),
+    PURGE_HISTORY_THREE_HOURS(3),
+    PURGE_HISTORY_SIX_HOURS(6),
+    PURGE_HISTORY_TWELVE_HOURS(12),
+    PURGE_HISTORY_TWENTY_FOUR_HOURS(24);
```

Comment:
@surajkumar you suggested to change enum name to `PurgeDuration` , then imo it would be better to rename all these var names to  something like `SIX_HOURS` instead of `PURGE_HISTORY_SIX_HOURS`.  That way when accessing these values, it would look like `PurgeDuration.SIX_HOURS` instead of `PurgeDuration.PURGE_HISTORY_SIX_HOURS`.

Reply:
Yeah, I would also suggest renaming the entries in that case.